Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -233,6 +266,11 @@ var NavigatorIOS = React.createClass({
       rightButtonTitle: PropTypes.string,
 
       /**
+       * If set, the right header button will appear with this system icon
+       */
+      rightButtonSystemItem: PropTypes.oneOf(Object.keys(SystemItems)),
```

Comment:
This should be `rightButtonSystemIcon` for consistency with the TabBarIOS API.

(It's not great that we used "icon" originally, but naming consistency is more important than correctness).